parent/pom.xml

```diff
@@ -1675,25 +1654,62 @@
       </build>
     </profile>
 
-    <!-- profile to disable the fmt-plugin on unsupported jdks -->
+    <!-- profile to auto format -->
     <profile>
-      <id>fmt-plugin</id>
+      <id>autoFormat</id>
       <activation>
-        <jdk>[,9)</jdk>
+        <!-- the autoFormat property is true by default, but seems to not be read here, so we need
+        to activate this by default specifically -->
+        <activeByDefault>true</activeByDefault>
```

Comment:
:upside_down_face: Profiles active by default are only active if no other profile is active. So when you specify a profile, e.g. `-Pprepare-offline`, it will automatically disable any `activeByDefault` profiles.